tests/parens/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -2407,7 +2407,9 @@ new Translator(
     new MessageFormatter(),
     'en',
     $someOtherVar,
-    array('foo' => 'bar')
+    array(
+        'foo' => 'bar'
+    )
```

Comment:
Should be collapsed, but it is problem with inline printing, we fix it in future, i think in next release we solve problem with printing `inline` nodes and release stable version

Reply:
Alright! :+1: